### ISSUE TITLE
Offset-anchor optimization technique in POI Manger

### ISSUE THREAD

**Issue by drogenlied** [Ulm-IQO/qudi#49]
Migration: trac#3.

It should be possible to re-optimize the alignment on a chosen POI by algorithmically going to a different POI and finding the current sample position. This would allow a POI to be tracked even when it is in a local fluorescence environment that is not suitable for direct optimization.

**Comment by Neverhorst** [comment]
I think this will be resolved by PR #366, right @kay-jahnke?

**Comment by kay-jahnke** [comment]
Yes, PR #366 resolves this one. I do however need to finish the PR first.
Gaussian fit onto the convolution in XY direction is still missing and turned our to be crucial.

**Comment by liuyanulm** [comment]
I believe this issue has been resolved.

**Comment by Neverhorst** [comment]
Actually this has not been resolved since PR #366 has been withdrawn without merging.
